### DISCUSSION TITLE
Notice of PPOCRLabel and StyleText Project Migration

### DISCUSSION THREAD

**Discussion by GreatV** [PaddlePaddle/PaddleOCR#13020] [General]
**关于PPOCRLabel和StyleText项目迁移的通知**

PPOCRLabel和StyleText项目已迁移到新的仓库。本仓库将专注于OCR和Structure。以下是详细信息：

1. **PPOCRLabel**：此项目已迁移到一个新的独立仓库 [PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)。

2. **StyleText**：同样地，StyleText项目也已迁移到新的独立仓库 [StyleText](https://github.com/PFCCLab/StyleText)。

---

**Notice of PPOCRLabel and StyleText Project Migration**

The PPOCRLabel and StyleText projects have been migrated to new repositories. This repository will now focus on OCR and Structure. Here are the details:

1. **PPOCRLabel**: This project has been migrated to a new standalone repository [PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel).

2. **StyleText**: Similarly, the StyleText project has also been migrated to a new standalone repository [StyleText](https://github.com/PFCCLab/StyleText).

**Comment by H4Njx** [comment]
> **关于PPOCRLabel和StyleText项目迁移的通知**
> 
> PPOCRLabel和StyleText项目已迁移到新的仓库。本仓库将专注于OCR和Structure。以下是详细信息：
> 
> 1. **PPOCRLabel**：此项目已迁移到一个新的独立仓库 [PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)。
> 2. **StyleText**：同样地，StyleText项目也已迁移到新的独立仓库 [StyleText](https://github.com/PFCCLab/StyleText)。
> 
> **Notice of PPOCRLabel and StyleText Project Migration**
> 
> The PPOCRLabel and StyleText projects have been migrated to new repositories. This repository will now focus on OCR and Structure. Here are the details:
> 
> 1. **PPOCRLabel**: This project has been migrated to a new standalone repository [PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel).
> 2. **StyleText**: Similarly, the StyleText project has also been migrated to a new standalone repository [StyleText](https://github.com/PFCCLab/StyleText).

hi! setup.py files lose in project [https://github.com/PFCCLab/PPOCRLabel](url)

**Comment by GreatV** [comment]
@H4Njx now it use `pyproject.toml`

**Comment by H4Njx** [comment]
> @H4Njx now it use `pyproject.toml`

thanks a lot! so please update the readme_ch.md  https://github.com/PFCCLab/PPOCRLabel/blob/main/README_ch.md#123-%E6%9C%AC%E5%9C%B0%E6%9E%84%E5%BB%BAwhl%E5%8C%85%E5%B9%B6%E5%AE%89%E8%A3%85

**Comment by GreatV** [comment]
@H4Njx update in
- https://github.com/PFCCLab/PPOCRLabel/pull/11

**Comment by H4Njx** [comment]
> @H4Njx update in
> 
> * [update readme.md PFCCLab/PPOCRLabel#11](https://github.com/PFCCLab/PPOCRLabel/pull/11)

thanks !!!!  I just want to create new branch and update the readme. For developers who have previously packaged python projects, this is not an issue, but it can cause great inconvenience for beginners.
